ex03.ex

```diff
@@ -54,12 +54,30 @@ defmodule Ex03 do
       iex> Ex03.odd_even [ 1, 2, 4, 7, 9 ]
       [ :odd, :even, :even, :odd, :odd ]
 
+      iex> Ex03.odd_even [ 3, 5, 7 ]
+      [ :odd, :odd, :odd ]
+
+      iex> Ex03.odd_even [ 2, 12 ]
+      [ :even, :even ]
+
+      iex> Ex03.odd_even [ 0 ]
+      [ :even ]
+
+      iex> Ex03.odd_even [ ]
+      [ ]
+
   (The library functions `Integer.is_even` and `Integer.is_odd` may
   be used if needed.)
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even([]), do: []
```

Comment:
# -1

I'm being a little harsh here, but it's because you are obviously getting this.

In your main odd_even function, you're doing two things: you are traversing the list and you're also converting a number into one of two atoms. That latter code is a transformation, and that's what functions do.

When you find yourself writing functions with two or more purposes, consider splitting them each purpose into its own transformation.